packages/flame/test/components/router_component_test.dart

```diff
@@ -4,6 +4,47 @@ import 'package:flame_test/flame_test.dart';
 import 'package:flutter/widgets.dart' hide Route, OverlayRoute;
 import 'package:flutter_test/flutter_test.dart';
 
+class _TestRoute extends Route {
```

Comment:
Could you please move this helper class to the bottom of the file? That way, when you open file you can immediately see `void main()` and can press the green arrow to start the tests.

Reply:
Done